### PR TITLE
✨ [+feature] Added option to sign artifacts during CI workflows

### PR DESCRIPTION
**Pull request by davidbrownell**
## :pencil: Description
Added option to sign artifacts during CI workflows.

## :gear: Work Item
https://dev.azure.com/gt-sse-center/Project%20Backlog/_workitems/edit/438

## :movie_camera: Demo
image
image

### CODE REVIEW

**Review by rfievet — DISMISSED** [review]
LGTM. Is the triple loop possible to be refactored? (in def Enumerate()

File: BootstrapEpilog.py [hunk 1]
```diff
@@ -13,7 +13,9 @@
 
 # First arg is the script name, second arg is the name of the shell script to write to
 for arg in sys.argv[2:]:
-    if arg == "--no-cache":
+    if arg == "--force":
```
[the remark this answers is not in the document]
Comment (davidbrownell) [reply BootstrapEpilog.py:16]: Good catch. This change was leftover after testing why builds failed due to a release change with micromamba. I will remove this change once your review is complete.

**Comment by davidbrownell** [comment]
> LGTM. Is the triple loop possible to be refactored? (in def Enumerate()

Those loops are calculating the permutations of the different flags, so I'm not sure how to improve it, but please let me know if you have ideas.

**Review by rfievet — APPROVED** [review]
LGTM

**Pull request merged**